app/services/preserved_object_handler.rb

```diff
@@ -32,29 +32,32 @@ def initialize(druid, incoming_version, incoming_size, endpoint)
     @logger = PreservationCatalog::Application.logger
   end
 
-  def create_after_validation
+  def create_after_validation(checksums_validated = false)
```

Comment:
I wonder whether it would be helpful to have more context here (maybe a comment?) about the fact that it is the *robots* who will have validated checksums, or not.